src/main/java/de/retest/recheck/ui/diff/Warning.java

```diff
@@ -0,0 +1,40 @@
+package de.retest.recheck.ui.diff;
+
+import javax.xml.bind.annotation.XmlAttribute;
+
+public class Warning {
```

Comment:
Oh and some Javadoc would be helpful, what is this warning about? I had to look into RET-1654 to get this information.